### PR TITLE
Adding models and Signup authentication and Database connection

### PR DESCRIPTION
**Pull request by adityabhagat007**

### CODE REVIEW

File: src/v1/controllers/auth/auth.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    const password = req.body.password;
+    const email = req.body.email;
+
+
+    const hash = await Bcryptjs.hash(password,12);
+    const user = new User({
+        name:name,
+        password: hash,
+        email:email
+    })
```
Comment (Bijoy-007) [inline src/v1/controllers/auth/auth.js:16]: - [ ] Mark the isVerified property as false or set a default value in your model

File: src/v1/controllers/auth/auth.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        const error = new Error('Something Went Wrong  please try again')
+        error.statusCode = 503;
+        throw error
+    } 
+
+    res.status(200).json({
+            message:"Success",
+            data:"One step Away to register yourself please check your email",
+            error:"",
+    })
```
Comment (Bijoy-007) [inline src/v1/controllers/auth/auth.js:29]: - [ ] Send the verification email before sending a successful response.
- [ ] send the error as null or empty object

**Review by Bijoy-007 — CHANGES_REQUESTED** [review]

**Comment by Bijoy-007** [comment]
@adityabhagat007 Make the necessary changes and open PR

File: src/v1/controllers/auth/auth.js (hunk 2 above)
Comment (adityabhagat007) [reply src/v1/controllers/auth/auth.js:29]: about to add otp verification

File: src/v1/controllers/auth/auth.js (hunk 1 above)
Comment (adityabhagat007) [reply src/v1/controllers/auth/auth.js:16]: okkk i will do that

**Pull request merged**